src/iterators/bfs.jl

```diff
@@ -0,0 +1,61 @@
+"""
+    BFSIterator
+
+`BFSIterator` is a subtype of [`VertexIterator`](@ref) to iterate through graph vertices using a breadth-first search. A source node(s) is optionally supplied as an `Int` or an array-like type that can be indexed if supplying multiple sources. If no source is provided, it defaults to the first vertex.
```

Comment:
Let me know if I'm wrong but the `BFSIterator` doesn't seem to allow multiple sources in the current form

Reply:
That is correct

Reply:
Now it does